readme.md

```diff
@@ -179,6 +179,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [SAWS](https://github.com/donnemartin/saws) - Supercharged AWS CLI.
 - [s3cmd](https://github.com/s3tools/s3cmd) - Fully-Featured S3 client.
 - [lstags](https://github.com/ivanilves/lstags) - Synchronize Docker images across different registries.
+- [ops](https://github.com/nanovms/ops/) - Unikernel Compilation and Orchestration Tool
```

Comment:
And, [as stated in the contribution guidelines](https://github.com/agarrharr/awesome-cli-apps/blob/master/contributing.md), move your addition to the end of the category.